src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -370,12 +371,14 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({ space }) => {
                 space.template as VenueTemplate
               ) && (
                 <AdminSection>
-                  <AdminCheckbox
-                    name="showShoutouts"
-                    label="Show shoutouts"
-                    variant="toggler"
-                    register={register}
-                  />
+                  <TesterRestricted>
```

Comment:
I don't think this change is needed any more. I would love to see the schema changes committed next week though.